Sources/Extensions/Shared/ColorExtensions.swift

```diff
@@ -255,7 +255,7 @@ public extension Color {
 // MARK: - Initializers
 public extension Color {
 	
-	/// SwifterSwift: Create NSColor from RGB values with optional transparency.
+	/// SwifterSwift: Create Color from RGB values with optional transparency.
```

Comment:
So basically this is the change we're looking for

Reply:
@SD10 thanks for the clarification. I'll go back and adjust this and submit a new pull request.

Reply:
@DannyMac180 No need for a new pull request. You can just make another commit changing them

Reply:
@SD10 thanks for your help! Maybe you can tell, this is my first attempt at contributing to an open source project.

Reply:
No problem @DannyMac180. You'll be a pro in no time 😁 👍 Thank you for submitting this fix 💯